src/proto/FunctionRpc.proto

```diff
@@ -391,6 +394,32 @@ message RpcLog {
   string properties = 7;
 }
 
+message RpcSystemLog {
+  enum Level {
+    Information = 0;
```

Comment:
All fields here apply for both system and invocation logs.  Possible to create a base type that can be resused for system logs and invocation logs? If that would be breaking change, we have to add this behind capabilities flags

Reply:
ping @yojagad

Reply:
@pragnagopa So, with the latest updates to https://github.com/Azure/azure-functions-nodejs-worker/pull/239 , I reused RpcLog as is (I'm using category to distinguish as you suggested). If that looks good, these set of proto changes may not be needed anymore and I can abandon this PR.